metadata.rb

```diff
@@ -6,3 +6,5 @@
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version           "2.1.2"
 recipe            "aws", "Installs the right_aws gem during compile time"
+
+depends "xfs"
```

Comment:
Since the xfs cookbook only installs xfsdump we can remove this and leave it up to the user to have a system capable of formatting and XFS volume